src/lennybot/lennybot.py

```diff
@@ -4,7 +4,7 @@
 import subprocess
 from datetime import datetime
 
-from git import Repo, GitDB
+from git import GitDB, Repo
```

Comment:
from unclear linting optimization this is always changed by isort but causes an error message with sonarlift afterwards. Manually changing it like suggested is the best practise imho.

```suggestion
from git import GitDB, Repo
```

Reply:
Sonatype lift is deactivated, therefore this is not needed